src/main/resources/hudson/plugins/postbuildtask/PostbuildTask/config.jelly

```diff
@@ -21,21 +21,22 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
 -->
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:block>
   <d:taglib uri="local">
     <f:entry title="${%Tasks}">
       <f:repeatable var="t" items="${instance.tasks}" add="Add another task" minimum="1">
-        <table width="100%" >
+        <div>
```

Comment:
if you want to update it like this it will break old versions of jenkins, you should update the baseline to 2.264 (jenkins.version property in the pom if doing this approach)

the normal approach that you can use to make it compatible either way cab be seen here:
[`85376de` (#33)](https://github.com/jenkinsci/urltrigger-plugin/pull/33/commits/85376de9521e5f3edcc5c9b9d13efa476a21d5c3)

Reply:
To elaborate, this recipe is documented on https://www.jenkins.io/doc/developer/views/table-to-div-migration/